x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -45,6 +45,23 @@ export class Simulator {
    */
   private readonly sideEffectSimulator: SideEffectSimulator;
 
+  /**
+   * An `enzyme` supported CSS selector for process node elements.
+   */
+  public static nodeElementSelector({
```

Comment:
This was a module level function before. Adding it as a static field allows simulator tests to access it. This way they can use `find` on returned `enzyme` wrappers to find graph node elements.